.github/workflows/main.yml

```diff
@@ -220,9 +253,26 @@ jobs:
       - name: Load agency image from cache
         run: docker load < /tmp/imgcache/img_agency.rar
 
-      - name: Verify image was loaded
+      - name: Verify agency image was loaded
         run: docker image ls --format "{{.Repository}}:{{.Tag}}" | grep "$CACHED_DOCKER_IMG_NAME_AGENCY" || { echo "Image $CACHED_DOCKER_IMG_NAME_AGENCY was not found!"; docker image ls; exit 1; }
 
+      - name: Load redis image cache
+        id: load-cached-image-redis
+        uses: actions/cache@v2
+        with:
+          path: /tmp/imgcache
+          key: ${{ env.CACHE_KEY_REDIS }}
```

Comment:
I wonder if this step is even necessary, given the whole cache file was already loaded in previous step?

Reply:
It is, because this is different cache - previous one is based on `CACHE_KEY_AGENCY` this one is based on `CACHE_KEY_REDIS`.

However I've realized that there might be some undesired effects as they both use the the same directory `/tmp/imgcache` - so instead I've distinguish cache directories to avoid any conflits. 

This link to your next comment suggesting to distinguish `image-redis.rar` - since I became now more explicit in naming cache directories, the file names probably doesn't matter and I call everything `image.rar`

Reply:
I am not sure, maybe keeping the same directory name would be better in case cache action overwrites the directory, since it saves storage on the runner? 🤔

Reply:
I am not sure, maybe keeping the same directory name would be better in case cache action overwrites the directory, since it saves storage on the runner? 🤔

Reply:
Does it overwrite it? I don't think so, but besides we are far from running out of storage in this CI.